apps/charge-accounts-service/src/main.ts

```diff
@@ -21,8 +22,12 @@ async function bootstrap () {
   }
 
   app.use(Helmet())
+  app.use(cookieParser())
   app.setGlobalPrefix('accounts')
-  app.enableCors()
+  app.enableCors({
```

Comment:
@MusabShakeel576 Please:

- Replace hardcoded origin strings with environment variables
- Use ConfigService to access these new environment variables

Reply:
@NikolayRn updated